theano/gpuarray/tests/test_dnn.py

```diff
@@ -2373,6 +2373,14 @@ def _meshgrid(height, width):
     img_out_gpu, = st_dnn_func(img, transform)
     img_out = np.asarray(img_out_gpu)
 
+    t_dy = T.tensor4('dy')
+    img_grad = T.grad(None, wrt=[t_img, t_theta], known_grads={st_dnn: t_dy})
+
+    grad_fn = theano.function([t_img, t_theta, t_dy], img_grad)
+
+    dy = -1 + 2 * np.random.randn(*img.shape).astype(theano.config.floatX)
+    spatialtf_grad = grad_fn(img, transform, dy)
```

Comment:
Hello @notoraptor , I've implemented the initial Python part of the gradients for the spatial transformer. I couldn't get further with the C implementation, because I got stuck in this point. The gradient function compiles, however when I try to execute it here, an error is shown (``theano.gof.utils.MethodNotDefined``), which is caused by the absence of the ``perform`` method in the Op.

Do you have any ideas on what could be the problem here?